packages/scrollbars/src/Scrollbar.js

```diff
@@ -0,0 +1,193 @@
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { withTheme, isRtl } from '@zendeskgarden/react-theming';
+import ScrollbarStyles from '@zendeskgarden/css-scrollbars';
+
+const COMPONENT_ID = 'scrollbars.scrollbar';
+
+class Scrollbar extends Component {
+  static propTypes = {
+    /**
+     * Enable dark styling
+     */
+    dark: PropTypes.bool,
+    active: PropTypes.bool,
+    scrolling: PropTypes.bool,
+    /**
+     * Reference to the react-custom-scrollbars component
+     */
+    scrollbarRef: PropTypes.func
+  };
+
+  static defaultProps = {
+    dark: false
+  };
+
+  state = {
+    isScrolling: false
+  };
+
+  /**
+   * Our `css-scrollbars` implementation provides these values for us
+   */
+  removeInvalidStyling = props => {
+    if (props.style) {
+      delete props.style.width;
+      delete props.style.height;
+      delete props.style.position;
+    }
+
+    return props;
+  };
+
+  /**
+   * react-custom-scrollbars requires a `ref` attribute which requires us to not use
+   * styled-components or a presentation component
+   */
+  renderHorizontalTrack = props => (
+    <div
+      {...this.removeInvalidStyling(props)}
+      className={classNames(
+        ScrollbarStyles['c-scrollbar__track'],
+        ScrollbarStyles['c-scrollbar__track--horizontal'],
+        {
+          [ScrollbarStyles['is-scrolling']]: this.isScrolling(),
+          [ScrollbarStyles['is-active']]: this.props.active
+        }
+      )}
+    />
+  );
+
+  /**
+   * react-custom-scrollbars requires a `ref` attribute which requires us to not use
+   * styled-components or a presentation component
+   */
+  renderVerticalTrack = props => (
+    <div
+      {...this.removeInvalidStyling(props)}
+      className={classNames(
+        ScrollbarStyles['c-scrollbar__track'],
+        ScrollbarStyles['c-scrollbar__track--vertical'],
+        {
+          [ScrollbarStyles['is-scrolling']]: this.isScrolling(),
+          [ScrollbarStyles['is-active']]: this.props.active
+        }
+      )}
+    />
+  );
+
+  /**
+   * react-custom-scrollbars requires a `ref` attribute which requires us to not use
+   * styled-components or a presentation component
+   */
+  renderHorizontalThumb = props => (
+    <div
+      {...this.removeInvalidStyling(props)}
+      className={classNames(
+        ScrollbarStyles['c-scrollbar__thumb'],
+        ScrollbarStyles['c-scrollbar__thumb--horizontal']
+      )}
+    />
+  );
+
+  /**
+   * react-custom-scrollbars requires a `ref` attribute which requires us to not use
+   * styled-components or a presentation component
+   */
+  renderVerticalThumb = props => (
+    <div
+      {...this.removeInvalidStyling(props)}
+      className={classNames(
+        ScrollbarStyles['c-scrollbar__thumb'],
+        ScrollbarStyles['c-scrollbar__thumb--vertical']
```

Comment:
This is not a real class (incorrect CSS documentation). Style is determined by the orientation of the parent (`.c-scrollbar__track--vertical`).